platform/engineering/design-docs/2020-12-10-auto-deploy-cms-build.md

```diff
@@ -71,21 +57,21 @@ To deploy content changes at will, that "auto-deploy" job is manually triggered
 
 - Content writers want to quickly draft, publish, and deploy content.
   - However, `vets-website` is only deployed to production once per day.
-  - The CMS team can run content-only deploys independently of the daily production deploy, but that is triggered manually.
+  - The CMS team can run content-only deploys independently of the daily production deploy, but that must be triggered manually.
 - The content-only deploy does not include automated accessibility checks.
 - In local development, engineers can encounter confusing build errors that are due to the mismatch in outdated locally cached content and updated Liquid templates.
 
 ### High Level Design
 
 There will be a new content-only auto-deploy job in addition to the existing full auto-deploy.
-- It will build content with CMS export data.
+- It will build content with GraphQL query data.
 - It will be a recurring job that executes on a more frequent schedule than the daily prod deploy.
 
 The initial schedule will run between 8am and 8pm ET to align with CMS support hours.
 - At first, it will be set to run hourly as a conservative cadence.
 - It will then be incrementally adjusted to a higher frequency as the build system allows.
 
-To avoid deploying when content hasn't changed since the previous deploy, the job will compare the export data (e.g., using and archiving the checksums of the tarballs).
+To avoid deploying when content hasn't changed since the previous deploy, the job will compare the GraphQL query data (e.g., using and archiving the checksums of the cached tarballs).
```

Comment:
I would remove this functionality for V1.0. It made more sense when we were doing an export. With graphQL we're just doing the same as the current only with hourly deploys.

Reply:
And when needed, I believe @indytechcook [mentioned](https://github.com/department-of-veterans-affairs/va.gov-team/pull/17304#discussion_r562649083) he could add an API endpoint with information of the most recent updated content.